test/e2e/testdata/acme/certificate_gcp.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cert-with-acme-dns01-clouddns-ambient
+spec:
+  secretName: cert-with-acme-dns01-clouddns-ambient
+  issuerRef:
+    # The issuer created previously
+    kind: ClusterIssuer
+    name: acme-dns01-clouddns-ambient
+    dnsNames:
+    - xxia-test.qe1.gcp.devcluster.openshift.com
+    - '*.xxia-test.qe1.gcp.devcluster.openshift.com'
```

Comment:
@geliu2016 To avoid failures again like https://issues.redhat.com/browse/OCPQE-16689 , we need to use random string instead of hard code in auto.

Reply:
updated.